types/bull/index.d.ts

```diff
@@ -170,6 +170,11 @@ declare namespace Bull {
 
     returnvalue: any;
 
+    /**
+     * Get progress on a job
+     */
+    progress(): any;
```

Comment:
This method signature is specified separately from `progress(value: any): Promise<void>` specifically so that the return type can be expressed as the synchronous result `any` and not `Promise<any>`, because the Job object doesn't return a resolved promise, it returns the actual value.